src/api/route-services/experiment.js

```diff
@@ -5,6 +5,26 @@ const {
 } = require('../../utils/dynamoDb');
 const NotFoundError = require('../../utils/NotFoundError');
 
+const getExperimentAttributes = async (tableName, experimentId, attributes) => {
```

Comment:
Extracted the calls to `dynamodb.getItem()` from the different service methods

Reply:
yeah I noticed, good idea 👍